spec/earning_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'earning'
+
+describe Earning do
+  let(:earning_amount) { 10 }
```

Comment:
This creates a var called `earning_amount`, but you then use the symbol `:earning_amount`.